python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -1958,6 +1958,19 @@ keys:
                         required: true
                         convert_types:
                           - str
+        traffic_engineering:
+          type: dict
+          keys:
+            enabled:
+              type: bool
+              description: Whether to enable traffic-engineering on this interface.
+            administrative_groups:
+              type: list
+              description: List of traffic-engineering administrative groups, valid values are names, ranges 0-127, or single integers 0-127.
```

Comment:
It support hexadecimal too.
```
traffic-engineering administrative-group ?
  <0x0-0xFFFFFFFF>,<0-127>,<0-127>-<0-127>,WORD  Administrative Group value in
                                                 hexadecimal, range or name
                                                 format
```

Reply:
Hexadecimals in yaml are interpreted as integers, which could lead to some confusion because if you input non string formatted hex 0xFFFF in your variables it will generate an integer value that is invalid in EOS = 65535. You might even have to do like "\"0xFFF\"" to have it translate all the way through. Also the hexadecimal entry type in EOS is functionally exactly the same as the range input and hex input is automatically translated to ranges in EOS so i think what i have here covers everything we need.

Reply:
```
arista(config-if-Et53)#traffic-engineering administrative-group 0xFFFF
arista(config-if-Et53)#show ac
interface Ethernet53
   no shutdown
   no switchport
   traffic-engineering administrative-group 0-15
```

Reply:
Yes you are right, in EOS it translate to ranges. We can skip to add the hexadecimal but @gmuloc @ClausHolbechArista can confirm.

Reply:
```
site1-wan1(config-s-42-if-Et3)#traffic-engineering administrative-group 0x42
site1-wan1(config-s-42-if-Et3)#show session-config diff
--- system:/running-config
+++ session:/42-session-config
interface Ethernet3
+   traffic-engineering administrative-group 1,6
site1-wan1(config-s-42-if-Et3)#traffic-engineering administrative-group 0x666
site1-wan1(config-s-42-if-Et3)#show session-config diff

--- system:/running-config
+++ session:/42-session-config
interface Ethernet3
+   traffic-engineering administrative-group 1-2,5-6,9-10
```

Looking at various outputs I am ok with not supporting hex, users can achieve the same with the range